patches/api/0419-add-getHighestMapID-method.patch

```diff
@@ -0,0 +1,23 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: chmodsayshello <chmodsayshello@hotmail.com>
+Date: Mon, 12 Jun 2023 18:33:43 +0200
+Subject: [PATCH] add getHighestMapID method
+
+
+diff --git a/src/main/java/org/bukkit/Server.java b/src/main/java/org/bukkit/Server.java
+index 1d1a1d087dabc9794e0062a064da2cced4062309..0f1469a6d89e340557e015ae1c1d370305c1ac9a 100644
+--- a/src/main/java/org/bukkit/Server.java
++++ b/src/main/java/org/bukkit/Server.java
+@@ -2161,5 +2161,12 @@ public interface Server extends PluginMessageRecipient, net.kyori.adventure.audi
+      * @return the potion brewer
+      */
+     @NotNull org.bukkit.potion.PotionBrewer getPotionBrewer();
++
++    /**
++     * Gets the highest map ID currently created
++     *
++     * @return the map ID (-1 if no maps were created yet)
++     */
++    int getHighestMapID();
```

Comment:
This is an incredibly small nitpick, but typically the `Id` naming convention is used in the Bukkit API, rather than the fully uppercase `ID`.

Reply:
done, thanks ;)